sim/midas/src/test/scala/midas/widgets/PlusargsBridgeSpec.scala

```diff
@@ -0,0 +1,61 @@
+// See LICENSE for license details.
+
+package goldengate.tests.widgets
+
+import chisel3._
+import midas.widgets._
+
+import org.scalatest._
+import org.scalatest.flatspec.AnyFlatSpec
+
+import chisel3._
+import chisel3.stage.ChiselStage
+import chisel3.experimental.BaseModule
+
+/** Unit tests for [[PlusargsBridge]]
+  */
+class PlusargsBridgeSpec extends AnyFlatSpec {
+
+  class PlusargsModuleIO(val params: PlusargsBridgeParams) extends Bundle {
+    val gotPlusargValue = Output(UInt((params.width).W))
+  }
+
+  // With the targetIO above, create a few simple misconfigurations by using
+  // only a subset of the IO
+  class CorrectInput(proto: PlusargsModuleIO) extends Bundle {}
+
+  class BridgeMock[T <: Bundle](gen: (PlusargsModuleIO) => T, params: PlusargsBridgeParams) extends BlackBox {
```

Comment:
In these tests you might as well use the PlusargsBridge companion object methods directly, to test the same code path users will actually walk.